oidc_provider/tests/cases/test_token_endpoint.py

```diff
@@ -429,6 +432,44 @@ def do_refresh_token_check(self, scope=None):
         response = self._post_request(post_data)
         self.assertIn('invalid_grant', response.content.decode('utf-8'))
 
+        # Refresh token doesn't expires with default settings
+        post_data = self._refresh_token_post_data(response_dic2['refresh_token'], scope)
+        expiry_time = timezone.now() + timedelta(
+            seconds=24*60*60,
+        )
+        with patch('oidc_provider.models.timezone.now') as time_func:
+            time_func.return_value = expiry_time
+            response = self._post_request(post_data)
+            self.assertIn('refresh_token', response.content.decode('utf-8'))
+
+    @override_settings(OIDC_IDTOKEN_INCLUDE_CLAIMS=True)
+    @override_settings(OIDC_REFRESH_TOKEN_EXPIRE=24 * 60 * 60)
+    def test_refresh_token_expiry(self, scope=None):
+        """
+        Refresh token expiry
+
+        Make sure that refresh token expires properly.
+        """
+        # Retrieve refresh token
+        code = self._create_code()
+        self.assertEqual(code.scope, ['openid', 'email'])
+        post_data = self._auth_code_post_data(code=code.code)
+        response = self._post_request(post_data)
+
+        response_dic1 = json.loads(response.content.decode('utf-8'))
+
+        # Use refresh token to obtain new token
+        post_data = self._refresh_token_post_data(
+            response_dic1['refresh_token'], scope
+        )
+        current_time = timezone.now() + timedelta(
+            seconds=24 * 60 * 60,
+        )
+        with patch('oidc_provider.models.timezone.now') as time_func:
+            time_func.return_value = current_time
+            response = self._post_request(post_data)
+            self.assertIn('invalid_token', response.content.decode('utf-8'))
+
```

Comment:
Would be better to have another test, where `OIDC_REFRESH_TOKEN_EXPIRE` is set, but the token has not expired yet

Reply:
Fixed.